src/views/connect_dialog_handler.rs

```diff
@@ -262,7 +266,7 @@ impl ConnectDialog {
                 self.nodeid.clear();
                 #[cfg(feature = "iroh")]
                 self.relay_url.clear();
-                Command::none()
+                text_input::focus(TCP_INPUT_ID.clone())
```

Comment:
for lines above, do you want/need to clear the other values entered when you switch tabs?
As long as the tab active when COnnect is pressed is the one used in the request - it would all work OK, no?
I know this is previous behaviour but only seen now...

Reply:
Yes, unless not closed the dialog will not clear the values